src/main/resources/static/css/wisvch-header.css

```diff
@@ -237,5 +237,4 @@ body.menu-open .page-header {
 .page-title .column h1, .page-title .columns h1, .page-title .col h1 {
     display: table-cell;
     vertical-align: middle;
-    padding: 0 10rem
 }
```

Comment:
Super picky, but you're missing an eof newline ;)

Reply:
ok!